main.go

```diff
@@ -57,6 +57,7 @@ var (
 	license   = flag.String("l", "apache", "license type: apache, bsd, mit, mpl")
 	licensef  = flag.String("f", "", "license file")
 	year      = flag.String("y", fmt.Sprint(time.Now().Year()), "copyright year(s)")
+	copyright = flag.String("x", "(c)", "custom copyright string used to determine license header presence")
```

Comment:
By defining `(c)` as the default value of the flag, it gets added as a copyright header for anyone who doesn't specify the flag.  Empty string is probably a better default, but don't do a Contains check on empty string, because all arrays contain a zero-byte subarray.

Is there a reason you chose `x` as the flag name?  I can't think of a mnemonic.  A full-word flag name is probably clearer.